composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "jimdo/prometheus_client_php",
+    "name": "imj/prometheus_client_php",
```

Comment:
I have some custom changes that depend on packagist, and actually I don't want to merge these changes, later I will rollback this commit. Sorry.